metashare/api/model_mixins.py

```diff
@@ -47,31 +51,38 @@ class PushMixin:
     Expects the following attributes:
         push_update_type: str
         push_error_type: str
-        get_serialized_representation: Callable[self]
+        get_serialized_representation: Callable[self, Optional[User]]
     """
 
+    def _create_context_with_user(self, user):
+        return {
+            "request": Request(user),
+        }
+
     def _push_message(self, type_, message):
+        """
+        type_:
+            str indicating frontend Redux action.
+        message:
+            {
+                "originating_user_id": str,
+                Optional["message"]: str  // error message
+            }
+        """
         async_to_sync(push.push_message_about_instance)(
             self, {"type": type_, "payload": message}
         )
 
     def notify_changed(self, *, type_=None, originating_user_id):
         self._push_message(
             type_ or self.push_update_type,
-            {
-                "originating_user_id": originating_user_id,
-                "model": self.get_serialized_representation(),
-            },
+            {"originating_user_id": originating_user_id},
         )
 
     def notify_error(self, error, *, type_=None, originating_user_id):
         self._push_message(
             type_ or self.push_error_type,
-            {
-                "originating_user_id": originating_user_id,
-                "message": str(error),
-                "model": self.get_serialized_representation(),
```

Comment:
@wlonk I think the `model` also should be removed from `report_scratch_org_error` in `push.py`?